src/collective/cover/testing.py

```diff
@@ -161,12 +153,6 @@ def setUpZope(self, app, configurationContext):
             manage_addVirtualHostMonster(app, 'virtual_hosting')
 
     def setUpPloneSite(self, portal):
-        if IS_PLONE_5:
-            self.applyProfile(portal, 'plone.app.contenttypes:default')
-        else:
-            if DEXTERITY_ONLY:
-                self.applyProfile(portal, 'plone.app.contenttypes:default')
-
```

Comment:
You aren't applying the profile of ```plone.app.contenttypes```.

Reply:
you don't need to is already applied above:

```diff
-    # this environment variable is set in .travis.yml test matrix
-    DEXTERITY_ONLY = os.environ.get('DEXTERITY_ONLY') is not None
+    from plone.app.contenttypes.testing import PLONE_APP_CONTENTTYPES_FIXTURE as PLONE_FIXTURE
+    DEXTERITY_ONLY = True
``

Reply:
Nice! But it's changing the portal's default workflow to [simple_publication_workflow](https://github.com/plone/plone.app.contenttypes/blob/1.1.x/plone/app/contenttypes/testing.py#L64), which is causing permission errors. However, when we create a portal the workflow is ```simple_publication_workflow```. Maybe it's the cover tests that are wrong :cry: .

Reply:
I think the problem may be in the tests or in the view permissions.